stockAndPrices/services/app/README.MD

```diff
@@ -0,0 +1,9 @@
+Python version : 3.79
+
+"pip install flask" in command line to download flask
+
+
```

Comment:
Leave a comment on how to start up flask server

Reply:
Updated Readme on how to start the server!